dist/index.html

```diff
@@ -0,0 +1,276 @@
+          <li class="nav__item">
+            <a href="#" class="nav__link" data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cardContainer">
+      <div class="card" data-qa="card">
+        <img class="card__image" src="/imac.7817668a.jpeg" alt="imac image">
+        <h3 class="card__prodName">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+        <p class="card__prodSku">Код товара: 195434</p>
+        <div class="starsPlusReviews">
+          <div class="starsPlusReviews__stars">
+            <div class="stars starsPlusReviews__stars--5">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="starsPlusReviews__prodReviews">Отзывов: 5</div>
+        </div>
+        <div class="price">
+          <div class="price__tag">Цена:</div>
+          <div class="price__value">69 999 грн</div>
+        </div>
+        <button class="cardButton">
+          <a href="#" data-qa="card-hover" class="cardButton__link">
+            КУПИТЬ
+          </a>
+        </button>
+      </div>
+      <div class="card" data-qa="card">
+        <img class="card__image" src="/imac.7817668a.jpeg" alt="imac image">
+        <h3 class="card__prodName">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+        <p class="card__prodSku">Код товара: 195434</p>
+        <div class="starsPlusReviews">
+          <div class="starsPlusReviews__stars">
+            <div class="stars starsPlusReviews__stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="starsPlusReviews__prodReviews">Отзывов: 5</div>
+        </div>
+        <div class="price">
+          <div class="price__tag">Цена:</div>
+          <div class="price__value">69 999 грн</div>
+        </div>
+        <button class="cardButton">
+          <a href="#" data-qa="hover" class="cardButton__link">
+            КУПИТЬ
+          </a>
+        </button>
+      </div>
```

Comment:
Without empty blank lines, it's much harder to read your code, so add blank lines. I mean this rule:
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/52370890/168752639-25f4da93-38e0-4695-b0f6-598cdcaf8c20.png">